ios/RCCLightBox.m

```diff
@@ -48,10 +48,27 @@ -(instancetype)initWithFrame:(CGRect)frame params:(NSDictionary*)params
                 UIColor *backgroundColor = [RCTConvert UIColor:style[@"backgroundColor"]];
                 if (backgroundColor != nil)
                 {
-                    self.overlayColorView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, frame.size.width, frame.size.height)];
+                    self.overlayColorView = [[UIView alloc] init];
                     self.overlayColorView.backgroundColor = backgroundColor;
                     self.overlayColorView.alpha = 0;
+                    [self.overlayColorView setTranslatesAutoresizingMaskIntoConstraints:NO];
                     [self addSubview:self.overlayColorView];
+
```

Comment:
While this will work, what do you think about just relaying out `self.overlayColorView` in `layoutSubviews` instead? This class already overrides it and most of RN uses manual view layout (because of using Yoga) instead of opting in to use autolayout.